examples/serverless-with-firebase/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "serverless-with-firebase",
+  "version": "1.0.0",
+  "description": "Next.js 8 serverless with firebase example",
+  "license": "MIT",
+  "engines": {
+    "node": "8"
+  },
+  "scripts": {
+    "dev": "next src/app",
+    "local": "run-s build local:serve",
+    "build": "run-s build:*",
+    "build:public": "cpx \"src/app/static/**/*.*\" dist/public/static -C",
+    "build:functions": "cpx \"src/functions/**/*.*\" dist/functions -C",
```

Comment:
I am working on a similar boilerplate. I ran into a problem where `-C` flag here would clear `dist/functions` if I run `build:app` beforehand. I ended up running the equivalent of `npm run clean` before doing my build commands instead of adding the `-C` flag to `cpx`.

Reply:
@johnapost `-C` option is doing the right thing. `run-s` will run commands in the order specified in package.json correctly, so the order matters. And I never run `build:app` along, you should always run `build` command.